addons/OPT/REVIVE/fn_clientInitEH.sqf

```diff
@@ -57,25 +57,24 @@ DFUNC(isUnconscious) =
 
 //var nach Respwan zurück setzen
 ["Respawn", {
-
-    {
-		player setVariable ["OPT_isUnconscious", 0, true];
-		player setVariable ["OPT_isStabilized", 0, true];
-		player setVariable ["OPT_isDragged", 0, true];
-		GVAR(OPT_isDragging) = false;
-		player allowDamage true;
-		1 enableChannel true;
-		1 fadeSound 1;
-		OPT_GELDZEIT_earplugsInUse = 1;
-		OPT_REVIVE_unconsciousHandler = nil;
-		OPT_REVIVE_respawnedHandler = nil;
-		player setVariable ["tf_unable_to_use_radio", false];
-
-		//Schaden Freigeben
-		player allowDamage true;
```

Comment:
Wurde das bewusst entfernt?

Reply:
War Doppelt schaue Zeile 69 (hehe)

Reply:
Aber die TFAR Variable nicht, oder?

Reply:
die ist auch noch da in Zeile 70. Hab die nur etwas sortiert um Globale Vars und Objekt Vars mal zu trennen.